src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Transaction;
+
+public interface OperationStrategy {
+    void executeOperation(Transaction transaction);
```

Comment:
Also, I would actually rather move the execution and handling of the transaction itself to the `TransactionProcessor` judging by its name, thus leaving a this class only a single responsibility of providing a proper `OperationHandler` which will be used in the processor, so:
```suggestion
public interface OperationStrategy {
    OperationHandler getHandlerFor(Transaction transaction);
```